changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,32 @@ Cisco Iosxr Collection Release Notes
 .. contents:: Topics
 
 
+v1.0.4
+======
+
+Release Summary
+---------------
+
+- Rereleased 1.0.3 with updated changelog.
+
+v1.0.3
+======
+
+v1.0.2
+======
+
+Minor Changes
+-------------
+
+- Add changelogs for previous releases (https://github.com/ansible-collections/cisco.iosxr/pull/47)
```

Comment:
This is probably not required. This may have made its way because the changelog PR for previous releases had a changelog fragment in it?